[2025-10-14T04:41:09.244+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-14T04:41:09.275+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_transactions scheduled__2025-09-13T00:00:00+00:00 [queued]>
[2025-10-14T04:41:09.292+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_transactions scheduled__2025-09-13T00:00:00+00:00 [queued]>
[2025-10-14T04:41:09.292+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-10-14T04:41:09.309+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_load_group.load_transactions> on 2025-09-13 00:00:00+00:00
[2025-10-14T04:41:09.318+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1696) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-14T04:41:09.320+0000] {standard_task_runner.py:72} INFO - Started process 1704 to run task
[2025-10-14T04:41:09.320+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'elt-pipeline', 'extract_load_group.load_transactions', 'scheduled__2025-09-13T00:00:00+00:00', '--job-id', '589', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmp6424bzyi']
[2025-10-14T04:41:09.322+0000] {standard_task_runner.py:105} INFO - Job 589: Subtask extract_load_group.load_transactions
[2025-10-14T04:41:09.405+0000] {task_command.py:467} INFO - Running <TaskInstance: elt-pipeline.extract_load_group.load_transactions scheduled__2025-09-13T00:00:00+00:00 [running]> on host e348b991e0be
[2025-10-14T04:41:09.531+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt-pipeline' AIRFLOW_CTX_TASK_ID='extract_load_group.load_transactions' AIRFLOW_CTX_EXECUTION_DATE='2025-09-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-13T00:00:00+00:00'
[2025-10-14T04:41:09.532+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-14T04:41:20.577+0000] {load.py:30} INFO - Loaded 34 rows and 4 columns to gcpl-469311.datalabs_techtest.stg_transactions
[2025-10-14T04:41:20.578+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-14T04:41:20.591+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-14T04:41:20.592+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=elt-pipeline, task_id=extract_load_group.load_transactions, run_id=scheduled__2025-09-13T00:00:00+00:00, execution_date=20250913T000000, start_date=20251014T044109, end_date=20251014T044120
[2025-10-14T04:41:20.630+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-14T04:41:20.653+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-14T04:41:20.655+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
